### ISSUE TITLE
yaml missing from install_requires

### ISSUE THREAD

**Issue by willspearman** [pywbem/pywbem#460]
Using pywbem-master, during install I see:

```build_moftab.py: Error: Import of pywbem package failed: No module named yaml```

And then:

```
(pywbem-latest) vagrant@vagrant-ubuntu-trusty-64:~/share/pywbem-master$ python
Python 2.7.12 |Continuum Analytics, Inc.| (default, Jul  2 2016, 17:42:40)
[GCC 4.4.7 20120313 (Red Hat 4.4.7-1)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
Anaconda is brought to you by Continuum Analytics.
Please check out: http://continuum.io/thanks and https://anaconda.org
>>> import pywbem
Traceback (most recent call last):
   File "<stdin>", line 1, in <module>
   File "pywbem/__init__.py", line 45, in <module>
     from ._recorder import *
   File "pywbem/_recorder.py", line 27, in <module>
     import yaml
ImportError: No module named yaml
>>>
```

Looks like the recorder needs yaml, so yaml needs to be added to the standard installation requirements.

**Comment by andy-maier** [comment]
Will,
thanks forvfinding this.

The yaml module comes with the PyYAML package which is installed for development. It should be moced to install.

**Comment by andy-maier** [comment]
Fixed in PR #463, along with fixing the need for `ordereddict` in python 2.6.
